cloud/src/meta-service/meta_service_job.cpp

```diff
@@ -553,30 +590,23 @@ void process_compaction_job(MetaServiceCode& code, std::string& msg, std::string
         return;
     }
 
-    //==========================================================================
-    //                               Lease
-    //==========================================================================
-    if (request->action() == FinishTabletJobRequest::LEASE) {
-        if (compaction.lease() <= 0 || recorded_compaction->lease() > compaction.lease()) {
-            ss << "invalid lease. recoreded_lease=" << recorded_compaction->lease()
-               << " req_lease=" << compaction.lease();
-            msg = ss.str();
-            code = MetaServiceCode::INVALID_ARGUMENT;
-            return;
-        }
-        recorded_compaction->set_lease(compaction.lease());
-        auto job_val = recorded_job.SerializeAsString();
-        txn->put(job_key, job_val);
-        INSTANCE_LOG(INFO) << "lease tablet compaction job, tablet_id=" << tablet_id
-                           << " key=" << hex(job_key);
-        need_commit = true;
-        return;
+    bool abort_compaction = false;
+    if (recorded_job.has_schema_change() && request->action() == FinishTabletJobRequest::COMMIT &&
+        !check_compaction_input_verions(compaction, recorded_job)) {
+        SS << "Check compaction input versions failed in schema change. input_version_start="
+           << compaction.input_versions(0) << " input_version_end=" << compaction.input_versions(1)
```

Comment:
`SS` typo? it should be lowercase

Reply:
#define SS (ss << &__FILE__[get_file_name_offset(__FILE__)] << ":" << __LINE__ << " ")